src/js/view/floating/floating-controller.ts

```diff
@@ -237,13 +245,20 @@ export default class FloatingController {
         }
 
         this.disableFloatingUI();
```

Comment:
`disableFloatingUI` changes player classes which toggle animations that could trigger CLS (but we won't see the start of those because we're hiding the wrapper)